src/util/PathUtil.ts

```diff
@@ -35,6 +35,26 @@ export function joinFilePath(basePath: string, ...paths: string[]): string {
   return posix.join(windowsToPosixPath(basePath), ...paths);
 }
 
+/**
+ * Resolves a path to its absolute form.
+ * Absolute inputs will not be changed (except changing Windows to POSIX).
+ * Relative inputs will be interpreted relative to process.cwd().
+ *
+ * @param path - Path to check (POSIX or Windows).
+ *
+ * @returns The potentially changed path (POSIX).
+ */
+export function absoluteFilePath(path: string): string {
+  if (posix.isAbsolute(path)) {
+    return path;
+  }
+  if (win32.isAbsolute(path)) {
```

Comment:
Are we sure that doing these two checks on both posix and windows is safe?
Should we instead check the current system, and only apply the system-applicable check?
(Can't think of a case myself where this would not be safe though)

Reply:
I can't think of any situations where this might be an issue. For completeness I actually checked how the `isAbsolute` functions work, the `posix` one just checks if the first character is `/`. The win32 check if the first character is `/` or `\`, or if the string matches `/^[a-zA-Z]:[\\/]/` (so just having the win32 one would actually be enough 😄 ).

Reply:
Btw, the reason for not checking the current system is that both systems can handle both path types (in case that would be required for some reason).